build_scripts/appimage/AppRun

```diff
@@ -24,7 +29,19 @@ do
     fi
 done
 
+
 export PYTHONNOUSERSITE=1
-(PYTHONHOME="${APPDIR}/opt/python3.11" exec "${APPDIR}/opt/python3.11/bin/python3.11" "${APPDIR}/usr/src/monkey_island.py" $@)
+
+export run_island="${APPDIR}/opt/python3.11/bin/python3.11 ${APPDIR}/usr/src/monkey_island.py"
```

Comment:
Environmental variables are usually all in caps. https://google.github.io/styleguide/shellguide.html#constants-and-environment-variable-names

Reply:
Should be addressed by https://github.com/guardicore/monkey/pull/3955